util/makefile_project/Makefile.rules

```diff
@@ -1,5 +1,5 @@
 SRC=../src
-CXXFLAGS+=-I../../../include
+CXXFLAGS+=-I../include -std=c++14
```

Comment:
Well it's apparently set relative to the repository which we don't want, because we don't want people building things in some sub-directory of our git-checkout. So it makes more sense to be relative to a possibly created new directory. This is also in line with the documentation which says
> we could create a directory include parallel to src, copy the release version of SeqAn into it and then change the include path of the compiler to point to this directory (value ../include).